operations/commit_queue.go

```diff
@@ -86,6 +92,63 @@ func deleteItem() cli.Command {
 	}
 }
 
+func mergeCommand() cli.Command {
+	return cli.Command{
+		Name:  "merge",
+		Usage: "test and merge a feature branch",
+		Flags: addProjectFlag(
+			cli.StringFlag{
+				Name:  joinFlagNames(refFlagName, "r"),
+				Usage: "merge branch `REF`",
+				Value: "HEAD",
+			},
+			cli.StringFlag{
+				Name:  identifierFlagName,
+				Usage: "finalize a preexisting item with `ID`",
+			},
+			cli.BoolFlag{
+				Name:  pauseFlagName,
+				Usage: "wait to enqueue an item until finalized",
```

Comment:
This gives the option to not enqueue now so it will be possible to add modules with the id returned by `uploadPatches()` (using a yet-to-be-implemented CLI command) and then to run this again to finalize.

By default the item will be finalized (unlike for patches).